internal/scorecard/alpha/scorecard.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/version"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	Log = logrus.New()
+)
+
+type Options struct {
+	Config       Config
+	Selector     labels.Selector
+	List         bool
+	OutputFormat string
```

Comment:
Non-blocking since I know the larger set of work is still a WIP, but would it make sense to have the list and outputformat options be handled concern in the CLI?

We could do something like:
```go

encoder, err := scorecard.GetEncoder(os.Stdout, outputFormat)
if err != nil {
    return err
}

var testOutput []Test
if list {
  testOutput, err = scorecard.ListTests(opts)
} else {
  testOutput, err = scorecard.RunTests(opts)
}
if err != nil {
    log.Fatal(err)
}

return encoder.Encoder(testOutput)
```

Reply:
yeah, not sure, lets settle this in another story when things shake out a bit more.